documentation/manual/src/main/asciidoc/modules/configuration.asciidoc

```diff
@@ -527,6 +527,38 @@ https://docs.jboss.org/author/display/WFLY10/Class+Loading+in+WildFly[WildFly do
 
 WildFly will by default attempt to guess which Persistence Provider you need by having a look at the `provider` section of the `persistence.xml`.
 
+==== Enable support for JEE 8
+
+Hibernate OGM {hibernate-ogm-version} requires **CDI 2.0** and **JPA 2.2**, that belong to **JEE8** specification.
+Wildfly 12.x has a limited support to JEE8.
```

Comment:
> Wildfly 12.x has a limited support to JEE8

multiple grammar problems, and Wildfly should be "WildFly" :)

It should read : "WildFly 12 has limited support for Java EE8"